debuglogger/default.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package debuglogger
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+)
+
+type defaultEvent struct {
+	level  LogLevel
+	logger *log.Logger
+	fields []byte
+}
+
+func (e *defaultEvent) Msg(msg string) {
+	if len(msg) == 0 {
+		return
+	}
+
+	e.logger.Print("[", e.level.String(), "] ", msg, string(e.fields))
```

Comment:
This will have spaces around level IIUC, not sure it's intended. I think printf is more readable and idiomatic in Go since the format string should only be parsed once

Reply:
I think there won't be spaces (see the test https://github.com/corazawaf/coraza/pull/682/files/3b216855c045a339da6e4913ff6e32d486b51f75#diff-96c090f66bd8cc6e041ae99779950909795fe020b400cf92dacd62fc68e4f568R107) but if the parsing happens once I am up to using Printf